chart/values.yaml

```diff
@@ -28,4 +28,4 @@ registry:
   # For token-based logins, use
   #
   # $ echo '{"registrytoken":"9cbaf023786cd7"}' | base64
-  authtoken: changeme
+  authtoken: e30K
```

Comment:
"e30K" is the base64 equivalent of "{}". It's actually a very weird docker behaviour in which you NEED to send an empty auth token even on registries with no authentication (like minikube w/ an in-cluster registry). Sending a string like "changeme" just causes docker to freak out because it can't parse it into a json object.